tests/components/wake_on_lan/test_switch.py

```diff
@@ -1,17 +1,28 @@
 """The tests for the wake on lan switch platform."""
-import unittest
+import platform
+import subprocess
+
+import wakeonlan
 
 import homeassistant.components.switch as switch
-from homeassistant.const import STATE_OFF, STATE_ON
-from homeassistant.setup import setup_component
+from homeassistant.const import (
+    ATTR_ENTITY_ID,
+    SERVICE_TURN_OFF,
+    SERVICE_TURN_ON,
+    STATE_OFF,
+    STATE_ON,
+)
+from homeassistant.setup import async_setup_component
 
 from tests.async_mock import Mock, patch
-from tests.common import get_test_home_assistant, mock_service
-from tests.components.switch import common
+from tests.common import async_mock_service
 
 TEST_STATE = None
 
 
+send_magic_packet = Mock(return_value=None)
```

Comment:
Since we never want to send the magic packet, let's just always stub it out. 
```suggestion
@pytest.fixture
def mock_send_magic_packet():
    """Mock magic packet."""
    with patch('wakeonlan.send_magic_packet'):
        yield
```